std/haxe/Serializer.hx

```diff
@@ -379,8 +379,6 @@ class Serializer {
 				}
 				#end
 			case TClass(c):
-				if (useCache)
-					cache.pop();
```

Comment:
I don't know if I'm missing some really intricate detail here or if this is really just silly. Nothing seems to happen between this `pop` and the `push` operations below that could access the cache, so I don't see why this was here.